blog/_posts/2017-09-22-we-heart-summer-of-code.md

```diff
@@ -4,8 +4,7 @@ layout: post
 created_at: Friday 22 September 2017
 permalink: blog/2017-09-22-we-heart-summer-of-code
 current: blog
-author: Emma Deacon
-twitter: emmamdeacon
```

Comment:
hey @MarckK, as explained in the [contributing guidelines](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#creating-a-new-blog-post) we need to have a twitter handle in order to have the post properly. Could you add the twitter line again?

Reply:
done